homeassistant/components/matter/strings.json

```diff
@@ -286,6 +328,9 @@
       },
       "child_lock": {
         "name": "Child lock"
+      },
+      "evse_charging_switch": {
+        "name": "EVSE charging"
```

Comment:
Shall we rename this to "Enable charging" because this is an actor, not a sensor.